.circleci/config.yml

```diff
@@ -83,6 +83,11 @@ workflows:
           jdk_version: openjdk8
           lein_test_command: lein with-profile -dev,+ci,+refactor-nrepl do clean, test
           <<: *test_code_filters
+      - test_code:
+          name: "JDK 8 including refactor-nrepl, and parallel Eastwood linters"
+          jdk_version: openjdk8
```

Comment:
What about creating `openjdk8-parallel`,  `openjdk11-parallel` executors with a higher CPU count. so that parallel code would exercised in depth?

See `resource_class`: https://circleci.com/docs/2.0/configuration-reference/#docker-executor

Reply:
Go for i! I'm curious what do